apps/condo/domains/organization/utils/serverSchema/Organization.js

```diff
@@ -55,6 +55,78 @@ async function createAdminRole (context, organization, data) {
     })
 }
 
+async function createDefaultRoles (context, organization, data) {
```

Comment:
you it's better to move it inside `organization/utils/serverSchema`
and remove useless `createAdminRole`